### PR TITLE
drop "Redirect to a Shortened URL"

### LINKED ISSUE #53: Error mode for short interaction URL
§2.5.1.1 Redirect to an Arbitrary Shortened URL

> If the AS cannot shorten its response URL enough to fit in the requested size, the AS SHOULD return an error.

Or maybe just ignore this part of the interaction request?

### LINKED ISSUE #121: Redirect to an Arbitrary Shortened URL
I don't understand why this mode is needed?

Its also problematic having a boolean or a number as valid parameters for `interact.redirect`

### PR DESCRIPTION
**Pull request by aaronpk**
closes #121 and closes #53

### CODE REVIEW

File: draft-ietf-gnap-core-protocol.md [hunk 1]
```diff
@@ -1490,13 +1490,10 @@ a browser request.
 ~~~
 
 In this non-normative example, the RC is indicating that it can 
-display a [user code](#request-interact-usercode) and direct the RQ
-to an [arbitrary URL of maximum length](#request-interact-short) 255
-characters, but it cannot accept a callback.
+display a [user code](#request-interact-usercode), but it cannot accept a callback.
```
Comment (jricher) [inline draft-ietf-gnap-core-protocol.md:1493]: This example can keep the redirection piece, just not the shortened form.

```suggestion
display a [user code](#request-interact-usercode) and direct the RQ
to an [arbitrary URL](#request-interact-redirect) on a secondary
device, but it cannot accept a callback.
```

File: draft-ietf-gnap-core-protocol.md [hunk 2]
```diff
@@ -1490,13 +1490,10 @@ a browser request.
...
 
 ~~~
     "interact": {
```
Comment (jricher) [inline draft-ietf-gnap-core-protocol.md:1496]:
```suggestion
    "interact": {
        "redirect": true,
```

**Review by jricher — CHANGES_REQUESTED** [review]
The same change needs to be made in the appendix under "Secondary Device Interaction {#example-device}". 

One of these examples can keep the redirection as an option alongside the user code, just not the shortened form.

**Comment by aaronpk** [comment]
Thanks, I missed that!

**Review by jricher — APPROVED** [review]

**Pull request merged**